modules/system/controllers/Updates.php

```diff
@@ -185,19 +184,17 @@ protected function getPluginVersionFile($path, $filename)
         $contents = [];
 
         try {
-            $updates = Yaml::withProcessor(new VersionYamlProcessor, function ($yaml) use ($path, $filename) {
-                return (array) $yaml->parseFile($path.'/'.$filename);
-            });
+            $updates = (array) Yaml::parseFile($path.'/'.$filename);
 
             foreach ($updates as $version => $details) {
                 if (!is_array($details)) {
                     $details = (array)$details;
                 }
 
                 //Filter out update scripts
-                $details = array_values(array_filter($details, function ($string) use ($path) {
+                $details = array_filter($details, function ($string) use ($path) {
                     return !preg_match('/^[a-z_\-0-9]*\.php$/i', $string) || !File::exists($path . '/updates/' . $string);
-                }));
+                });
```

Comment:
@jaxwilko @mjauvin what was the reason for this change?

Reply:
We somehow got the Yaml preprocessor changes in the branch which was causing us all sorts of issues so I cherry picked the commit reverting it across:

`git cherry-pick 0ebe3bc07416e582c40e0e7c4f5737a08329db60`